widgetssdk/src/main/java/com/glia/widgets/callvisualizer/CallVisualizerSupportActivity.kt

```diff
@@ -1,5 +1,17 @@
 package com.glia.widgets.callvisualizer
 
+import android.os.Parcelable
 import androidx.appcompat.app.AppCompatActivity
+import com.glia.widgets.callvisualizer.controller.CallVisualizerController
+import kotlinx.parcelize.Parcelize
 
-class CallVisualizerSupportActivity : AppCompatActivity()
\ No newline at end of file
+class CallVisualizerSupportActivity : AppCompatActivity() {
+    companion object {
+        val PERMISSION_TYPE_TAG: String = CallVisualizerController::class.java.simpleName
```

Comment:
NIT: this should be `const`. With `const` it is possible to use without pointing to `Companion`

Reply:
We figured out that it's not possible for this case:

<img width="978" alt="Screenshot 2023-04-28 at 12 44 26" src="https://user-images.githubusercontent.com/57521863/235133277-86d37dd6-d13a-4ad4-85d5-c925a4810210.png">

Reply:
My bad